README.md

```diff
@@ -16,39 +16,68 @@ This version of the container includes:
 - pgTAP 1.3.3 (from theory/pgtap@02bc769c92c48d01e4c2f76db6523287017b45a9)
   - Support for PostgreSQL 9.6, 10, 11, 12, 13, 14, 15, and 16
 - pg_prove 3.36
-- Sqitch 1.4.0
+- Sqitch 1.4.1
 
 The version of pgTAP is installed and uninstalled as needed; unit test files
 _**must not**_ include `CREATE EXTENSION pgtap`.
 
 These images can be pulled either from Docker Hub
-(`kineticcafe/sqitch-pgtap:2.4`) or the GitHub Container Registry
-(`ghcr.io/kineticcafe/sqitch-pgtap:2.4`).
+(`kineticcafe/sqitch-pgtap:2.5`) or the GitHub Container Registry
+(`ghcr.io/kineticcafe/sqitch-pgtap:2.5`).
 
-## `run` script Commands
+## `kineticcafe-sqitch-pgtap` script Commands
 
-The `run` script is recommended for running everything as it manages environment
-variable configuration for each run. The `run` script will pull from
-`ghcr.io/kineticcafe/sqitch-pgtap:2` by default; this can be overridden by
-using `$IMAGE`:
+The `kineticcafe-sqitch-pgtap` script is recommended for running everything as
+it manages environment variable configuration for each run. The
+`kineticcafe-sqitch-pgtap` script will pull from
+`ghcr.io/kineticcafe/sqitch-pgtap:2` by default; this can be overridden by using
+`$IMAGE`:
 
 ```console
 $ IMAGE=kineticcafe/sqitch-pgtap:latest ./run version
 [gchr.io/]kineticcafe/sqitch-pgtap:2.5.0
 
-  alpine 3.18
-  sqitch (App::Sqitch) v1.4.0
+  alpine 3.19
+  sqitch (App::Sqitch) v1.4.1
   pgtap 1.3.1
   pg_prove 3.36
 ```
 
+### Installing `kineticcafe-sqitch-pgtap`
+
+`kineticcafe-sqitch-pgtap` can be installed with symlinks using the `install`
+script:
+
+```sh
+curl -sSL --fail \
+  https://raw.githubusercontent.com/KineticCafe/docker-sqitch-pgtap/main/install |
+  bash -s -- ~/.local/bin
```

Comment:
Is there any bash version requirement (thinking of how GPL has forced Apple to stay on an old version)

Reply:
I did some testing and it doesn't require anything other than Bash 3 (and would probably work with zsh, too). It should be changed to POSIX sh (`/bin/sh`), but there are features used (arrays, mostly) that have no equivalent in POSIX and I don't feel like dealing with that right now.